lib/v2/nua/utils.js

```diff
@@ -47,8 +47,15 @@ const ProcessFeed = async (items, artiContent, cache) =>
                 const $ = cheerio.load(result.data);
 
                 item.author = $('.arti_publisher').text() + '  ' + $('.arti_views').text();
-                item.description = $(artiContent).html();
 
+                const hotText = '阅读原文';
+                const url = item.link;
+
+                if (item.link.includes('weixin')) {
+                    item.description = `暂不支持解析微信公众号内容，请点击 ${hotText.link(url)}`;
```

Comment:
RSSHub has [utility](https://github.com/DIYgod/RSSHub/blob/master/lib/utils/wechat-mp.js) to process those articles